spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc

```diff
@@ -196,6 +196,173 @@ spring:
     driver-class-name:org.postgresql.Driver
 ----
 
+[start=4]
```

Comment:
We need to remove any references to statements like this:

> Since there is a Spring Cloud Data Flow Server for each target platform, you need to modify the appropriate maven `pom.xml` for each platform. 

This is right above the changes where you're attempting to make.

Reply:
Perhaps also change:

> There are tags in each GitHub repository for each server version.

to

> There are GA release tags in GitHub repository, so you can switch to desired GA tags to add the drivers on the production-ready codebase.